hosting/how-to-host-on-sia.md

```diff
@@ -1,151 +1,239 @@
 
-* **A reliable and stable setup:** You need a computer, internet connection, and electricity that you can trust. Hosts are required to be online nearly 100% of the time or face financial penalties.
+* **A reliable and stable setup:** You need a computer, Internet connection, and electricity that you can trust. Hosts are required to be online nearly 100% of the time or face financial penalties.
 * **Sia Prerequisites:** Make sure your computer meets the Sia requirements
-  * CPU: Sia does not require special CPU considerations
-  * RAM: 8 GB recommended
-  * SSD: 40 GB recommended \(keep an eye on your consensus size!\)
+  * CPU: Sia does not require special CPU considerations. However, your CPU may become a bottleneck if your host becomes very popular with the renters, and they start uploading or downloading a lot of data to or from your host.
+  * RAM: 8 GB is a minimum, although your host will then likely be running out of memory quite often; 16 or even 32 GB is better.
+  * OS drive: An SSD with minimum 40 GB of free space recommended. At the time of writing this article, Sia blockchain was about 32 GB in size, and it’s growing constantly. You should avoid letting your OS drive run out of space, because then you will risk corrupting your host’s metadata.
   * Create a Sia wallet
   * [Get Siacoins](../get-started-with-sia/how-to-buy-siacoins.md)
-* **Storage space:** Spare hard drive space to rent out, ideally greater than 4 TB.
+* **Storage space:** Spare hard drive space to rent out, ideally greater than 4 TB. It doesn’t have to be an SSD: an HDD would do just fine and also be much cheaper.
+* **Sia software:** You can choose between Sia UI \(GUI\) and Sia Client, download it [here](https://sia.tech/get-started).
 
-### Enable Disk Compression \(Windows only, Optional\)
-Windows does not support sparse files by default, which causes downtime when adding large storage folders. This downtime can be avoided by enabling disk compression on any disk that you use for Sia storage. 
+## Preparing Your Storage Drive\(s\) \(Windows Only, Optional\)
 
-![](../.gitbook/assets/host-disk-compression.png)
+NTFS file system of Windows does not support sparse files, which causes downtime when adding large \(several TB\) storage folders. To avoid this downtime, you can format your storage drive using ReFS before adding it to Sia network.
 
-## The process
+Unfortunately, ReFS is not available on all Windows releases. It is currently available on Windows 10 Server, Windows 10 Enterprise, and Windows 10 Pro for Workstations. Of these, Windows 10 Pro for Workstations is the most available solution: it’s very easy to upgrade from Windows 10 Pro, and the upgrade license doesn’t cost much. After all, you will probably earn more Siacoins within the time, which your host would otherwise be spending on filling your storage space with zeroes, like it happens on NTFS.
```

Comment:
Probably better to remove ", and the upgrade license doesn’t cost much" and "After all, you will probably earn more Siacoins within the time, which your host would otherwise be spending on filling your storage space with zeroes, like it happens on NTFS."  Cost is relative and the amount of earnings is unpredictable